.github/workflows/build-and-test.yml

```diff
@@ -188,8 +220,8 @@ jobs:
           command: test
           args: --manifest-path ./bindings/wasm/Cargo.toml --release
```

Comment:
Aside: shouldn't this step run with the `bindings/wasm` `working-directory` to avoid the `manifest-path`? I bring this up only because I wonder if the different working directory affects cache hits/artifact re-use between this and other steps.

Reply:
This is the full step (couldn't select it properly from the UI):

```yml
- name: Run tests
  uses: actions-rs/cargo@v1
  with:
    command: test
    args: --manifest-path ./bindings/wasm/Cargo.toml --release
```

Reply:
The `cargo` action doesn't offer a working-directory option. So I think we only solution would be to run a script instead. It might explain why the target cache isn't be hit, though. @cycraig what do you think, should we change it to script?

Reply:
I think it's definitely worth a try, we don't seem to depend on any of the advanced options of the `cargo` action so changing it to a `run` step should be equivalent (except with the right `working-directory`).